CONTRIBUTING.md

```diff
@@ -32,23 +39,23 @@ Our base of operations for group discussions on this project: [Jupiter Web Site
+* _Please do a quick search prior top opening an issue - maybe a relevant issue already exists_ 😉
+* 💡 When you feel inspired about a particular issue or feature, please propose your implementation idea/strategy for approval via the issue comments, which indicates you are dedicated to solving this issue, thus avoiding duplication of effort.
 * 💪 If you have a particular skillset, feel free to focus there.
-* 🚩 Some tickets are specifically [labeled](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/labels) _"low priority"_, but others are so far of fairly equal importance. Anytime there is something clearly urgent, the _"urgent"_ tag will be used.
+* 🚩 Some tickets are specifically [labeled](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/labels) _"low priority"_, but others are so far of fairly equal importance. Any time there is something clearly urgent, the _"urgent"_ tag will be used.
 
 
 ### How do I assign an issue to myself?
 
-* 🔑 The key is to communicate enough to avoid unnecessary duplication of efforts, which has already happened unfortunately
-* 🗨 Comment on an issue to signal your intentions and interest. The important thing is to communicate your intention and progress so we can help you find success. And, as we've seen already: it seems like when there's movement on a ticket we all get inspired aswell. Progress is wonderfully contagious! ✨
+* 🔑 The key is to communicate enough to avoid unnecessary duplication of effort, which has already happened, unfortunately.
+* 🗨 Comment on an issue to signal your intentions and interest. The important thing is to communicate your intention and progress so we can help you find success. And, as we've seen already: it seems like when there's movement on a ticket we all get inspired. Progress is wonderfully contagious! ✨
 
 ### What is the Pull-Request (PR) etiquette?
 
-* 💻 Don't be shy of creating a Work-In-Progress (WIP) PR. This is a great way to geat some feedback and maybe some direction on your work.
-  * ✅ Create a markdown checklist to indicate what's done and what's still missing ([see this example](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/112))
-* ⚙ If you're closing a specific issue I encourage you to use [Github's closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to automatically close the issue once it the PR is merged.
-* 🚮 Feel free to delete your feature branch after it has been merged via the PR. There’s a button handy on the GitHub interface, and worst case they can be reverted easily.
+* 💻 Don't be shy to create a Work-In-Progress (WIP) PR. This is a great way to get some feedback and maybe some direction for your work.
```

Comment:
I think I would drop to and add a comma after shy.

Reply:
Done